pallets/dkg/Cargo.toml

```diff
@@ -56,6 +56,10 @@ frost-p256 = { workspace = true, default-features = false }
 frost-p384 = { workspace = true, default-features = false }
 snowbridge-milagro-bls = { workspace = true, default-features = false }
 
+# WSTS specific dependencies
+wsts = "3.0.0"
```

Comment:
This needs no default features

Reply:
We should remove this nonetheless. Better to get things working against the existing `DigitalSignatureScheme` flow for `SchnorrP256`.